--- darksnow-0.6/Makefile.in	2005-10-26 16:12:18.000000000 +0000
+++ Makefile.in.debian	2005-10-26 16:17:20.000000000 +0000
@@ -22,12 +22,12 @@
 INTLPREFIX=@prefix@
 CC=@CC@
 VERSION=@VERSION@
-CFLAGS+=-Wall 
+CFLAGS+=-Wall -fPIC
 INSTALL=install
 ARCH=`uname -m`
 
 #set debugging symbols:
-#CFLAGS+= -g 
+CFLAGS+= -g 
 
 #set funny messages
 #CFLAGS+= -DFUNNY 
@@ -60,34 +60,12 @@
 	rm -rf slack darksnow.tgz
 
 install: darksnow
-	$(INSTALL) -D --mode=0755 darksnow $(PREFIX)/bin/darksnow
-	$(INSTALL) -D --mode=0644 documentation/CHANGES $(PREFIX)/share/darksnow/changelog
-	$(INSTALL) -D --mode=0644 po/pt_BR.mo $(PREFIX)/share/locale/pt_BR/LC_MESSAGES/darksnow.mo
-	$(INSTALL) -D --mode=0644 icon/darksnow_icon.png $(PREFIX)/share/icons/hicolor/32x32/apps/darksnow.png
-	$(INSTALL) -D --mode=0644 man/darksnow.1.gz $(PREFIX)/share/man/man1/darksnow.1.gz
-
-make_pkg: darksnow
-	echo "Remember to run this option as root!"
-	rm -rf slack
-	$(INSTALL) -D --mode=0755 darksnow slack$(PREFIX)/bin/darksnow
-	$(INSTALL) -D --mode=0644 icon/darksnow_icon.png slack$(PREFIX)/share/icons/darksnow.png
-	$(INSTALL) -D --mode=0644 po/pt_BR.mo slack$(PREFIX)/share/locale/pt_BR/LC_MESSAGES/darksnow.mo
-	$(INSTALL) -D --mode=0644 documentation/CHANGES slack$(PREFIX)/doc/darksnow/CHANGES
-	$(INSTALL) -D --mode=0644 documentation/LICENSE slack$(PREFIX)/doc/darksnow/LICENSE
-	$(INSTALL) -D --mode=0644 documentation/README.en slack$(PREFIX)/doc/darksnow/README.en
-	$(INSTALL) -D --mode=0644 documentation/README.pt_BR slack$(PREFIX)/doc/darksnow/README.pt_BR	
-	$(INSTALL) -D --mode=0644 documentation/CHANGES slack$(PREFIX)/doc/darksnow/CHANGES
-	$(INSTALL) -D --mode=0644 documentation/slack-desc slack/install/slack-desc
-	cd slack; makepkg -c y -l n  ../darksnow-$(VERSION)-$(ARCH)-1rd.tgz
-	echo "Remember to run this option as root!"
-
-install_pkg: make_pkg
-	/sbin/installpkg darksnow-*-1rd.tgz
-
-upgrade_pkg: make_pkg
-	/sbin/upgradepkg darksnow-*-1rd.tgz
+	$(INSTALL) -D --mode=0755 $(CURDIR)/darksnow $(DESTDIR)$(PREFIX)/bin/darksnow
+	$(INSTALL) -D --mode=0644 $(CURDIR)/documentation/CHANGES $(DESTDIR)$(PREFIX)/share/doc/darksnow/changelog
+	$(INSTALL) -D --mode=0644 $(CURDIR)/po/pt_BR.mo $(DESTDIR)$(PREFIX)/share/locale/pt_BR/LC_MESSAGES/darksnow.mo
+	$(INSTALL) -D --mode=0644 $(CURDIR)/icon/darksnow_icon.png $(DESTDIR)$(PREFIX)/share/icons/hicolor/32x32/apps/darksnow.png
+	$(INSTALL) -D --mode=0644 $(CURDIR)/man/darksnow.1.gz $(DESTDIR)$(PREFIX)/share/man/man1/darksnow.1.gz
 
 dist-clean: clean
 	rm aclocal.m4 config.status config.log man/darksnow.1.gz Makefile
